doc/cephfs/fs-nfs-exports.rst

```diff
@@ -36,6 +36,15 @@ Update NFS Ganesha Cluster
 
 This updates the deployed cluster according to the placement value.
 
+Delete NFS Ganesha Cluster
+==========================
+
+.. code:: bash
+
+    $ ceph nfs cluster delete <clusterid>
+
+This deletes the deployed cluster.
+
```

Comment:
what's the difference to `ceph orch rm nfs <clusterid>`?

Reply:
Currently nothing. But I will remove ganesha pool if we have only one cluster available.